corehq/apps/data_dictionary/models.py

```diff
@@ -91,3 +94,11 @@ def save(self, *args, **kwargs):
         from .util import get_data_dict_props_by_case_type
         get_data_dict_props_by_case_type.clear(self.case_type.domain)
         return super(CaseProperty, self).save(*args, **kwargs)
+
+    def valid_value(self, value):
```

Comment:
I'd call this `is_valid` to make it clearer that it's going to return a boolean.

Reply:
Changed it to `check_validity` since with the movement of importer exception raising into there this ended up not being a method that returns a boolean.